libraries/botbuilder-dialogs-declarative/src/resources/resourceExplorer.ts

```diff
@@ -215,4 +222,42 @@ export class ResourceExplorer {
             this._eventEmitter.emit(event, resources);
         }
     }
+
+    private load<T extends object>(value: Record<string, any>): T {
```

Comment:
Prefer `unknown` over `any` as it's more semantically correct and requires proper type checking/casting at runtime.